atk/src/lib.rs

```diff
@@ -1,10 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-//! # ATK bindings
+//! # Rust ATK bindings
 //!
-//! This library contains safe Rust bindings for [ATK](https://developer.gnome.org/atk/). It's
-//! a part of [Gtk-rs](https://gtk-rs.org/).
-
+//! This library contains safe Rust bindings for [ATK](https://developer.gnome.org/atk).
+//! It is a part of [Gtk-rs](https://gtk-rs.org/).
```

Comment:
Maybe it should mention it's part of gtk3-rs bindings instead of gtk-rs?

Reply:
We are talking about the organization there, not the repository (hence the URL).

Reply:
sure, what i meant is it's maybe worth it to mention that atk is part of the specific gtk3-rs bindings, which are part of gtk-rs.org.

Reply:
Not sure if this is worth it to mention it in the documentation. What do you think @sophie-h @sdroege ?

Reply:
then what mentioning that atk is part of gtk-rs.org brings to the docs? you will already be reading them from gtk-rs.org/gtk3-rs/ 
but it's just a suggestion, probably doesn't worth the change, feel free to ignore it

Reply:
I think there is too much room for improvement for all these missing introductory docs to start tweaking them now. Just wanted to have two sentences and a header everywhere.

Reply:
Indeed, never mind my comment :-)